src/hdf5/atms_netcdf_hdf5_2ioda.py

```diff
@@ -257,6 +260,28 @@ def get_WMO_satellite_ID(filename):
     return WMO_sat_ID
 
 
+def get_epoch_time(obs_time_utc):
+
+    # ugh this does not seem generic
+    year = obs_time_utc[:, :, 0].flatten()
```

Comment:
@fabiolrdiniz is this where you think the vectorize would be helpful, and was that a numpy call?

Reply:
Yes, that would be a `numpy.vectorize(FUNCTION_HANDLING_DATETIME)` call to avoid looping over the dates. But the code that you added here is doing something similar with `iterables`.